introduction.md

```diff
@@ -0,0 +1 @@
+My name is Iszlai Lorand-Edmond, i'm a lvl 10 samuray.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.